index.d.ts

```diff
@@ -37,17 +37,17 @@ export declare interface Epic<Input extends Action = any, Output extends Input =
   (action$: ActionsObservable<Input>, state$: StateObservable<State>, dependencies: Dependencies): Observable<Output>;
 }
 
-export interface EpicMiddleware<T extends Action, O extends T = T, S = void, D = any> extends Middleware {
+export interface EpicMiddleware<T extends Action, O extends T = T, S = any, D = any> extends Middleware {
```

Comment:
can we have a tighter type here?

is there no State type instead of a any type?